scripts/load-breaches.js

```diff
@@ -33,7 +33,7 @@ async function handleBreachesResponse(response) {
 
     for (const breach of breachesJSON) {
       // purify the description going into the DB
-      breach.Description = DOMPurify.sanitize(breach.Description);
+      breach.Description = DOMPurify.sanitize(breach.Description, {ALLOWED_TAGS: []});
```

Comment:
What is this change doing?
My gut is telling me it'd strip all the HTML tags in the description.

Reply:
Your gut is correct.

Reply:
But what about all the breaches that have links in the description, a la https://fx-breach-alerts.herokuapp.com/?breach=Adobe

Reply:
This change was done intentionally to make sure we don't have links in the description. The alternative would have been to de-link-ify stuff from the client side. FWIW, the text content of the links are preserved after sanitize() :)

Reply:
OK... but now I'm wondering why I'm still seeing links on https://fx-breach-alerts.herokuapp.com/?breach=Adobe because that means either the Heroku site isn't deploying properly, or something is cached, or the code isn't doing what we're expecting.

Reply:
Also, this will make some breaches a bit awkward. Like in the case of [7k7k Breach](https://fx-breach-alerts.herokuapp.com/?breach=7k7k):

> In approximately 2011, it's alleged that the Chinese gaming site known as [7k7k](http://www.7k7k.com/) suffered a data breach that impacted 9.1 million subscribers. Whilst there is evidence that the data is legitimate, due to the difficulty of emphatically verifying the Chinese breach it has been flagged as "unverified". The data in the breach contains usernames, email addresses and plain text passwords. [Read more about Chinese data breaches in Have I been pwned.](https://www.troyhunt.com/handling-chinese-data-breaches-in-have-i-been-pwned/)

Note that the last line will theoretically not have the actual text linked, so we'll have some confusing messaging saying "Read more about..." where the user cannot, in fact, read more.